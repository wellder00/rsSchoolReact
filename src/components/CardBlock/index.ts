export { default as CardBlock } from './CardBlock';
