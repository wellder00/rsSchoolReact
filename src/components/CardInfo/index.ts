export { default as CardInfo } from './CardInfo';
