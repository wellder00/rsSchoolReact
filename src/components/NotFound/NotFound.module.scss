.wrapper {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin-top: 30px;
  font-size: 30px;
  font-family: 'Metal Mania', sans-serif;
  letter-spacing: 5px;
}

.image {
  max-width: 400px;
}
