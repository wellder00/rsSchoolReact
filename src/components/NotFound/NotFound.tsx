import styles from './NotFound.module.scss';
import notFound from '../../assets/images/notFound.png';

const NotFound = () => {
  return (
    <div className={styles.wrapper}>
      <img className={styles.image} src={notFound} alt="Not Found" />
      <h2>Not Found</h2>
    </div>
  );
};

export default NotFound;
