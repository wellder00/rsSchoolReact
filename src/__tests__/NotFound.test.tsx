import { NotFound } from '../components/NotFound';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

test('should show loading indicator', () => {
  const { asFragment } = render(<NotFound />);
  expect(asFragment()).toMatchSnapshot();
  expect(screen.getByText(/not found/i)).toBeInTheDocument();
});
