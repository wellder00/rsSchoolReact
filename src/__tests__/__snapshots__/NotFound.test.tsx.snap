// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should show loading indicator 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <img
      alt="Not Found"
      class="image"
      data-nimg="1"
      decoding="async"
      height="100"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=640&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=384&q=75 1x, /_next/image?url=%2Fimg.jpg&w=640&q=75 2x"
      style="color: transparent;"
      width="300"
    />
    <h2>
      Not Found
    </h2>
  </div>
</DocumentFragment>
`;
